src/app/components/split-view/index.html

```diff
@@ -0,0 +1,239 @@
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-search-toggle.png" />
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>When possible, pre-filter the list to relevant items and pre-sort it to the expected useful order. If you can't predict how users will use the list, include a <a routerLink="/components/list/toolbar">list toolbar</a> to let them add items or filter, sort, and search the list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/option-toolbar.png" />
+            <div class="caption caption-warning sky-text-warning">Use caution when using a full toolbar with a split view. Automatically filter and sort the list when possible.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Behavior and states">
+      <h3>Automatically advance to the next item</h3>
+        <div class="sky-text-block">
+          <p>If users are likely to work through the list sequentially, automatically advance to the next item after users invoke the primary action in the workspace.</p>
+        </div>
+      <h3>Place focus in the workspace</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>When users select an item in the list, place focus on the first focusable element in the workspace so that users can start working immediately.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/behavior-focus.png" />
+          </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <h3>Change panel width</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Users can change the panel widths using drag-and-drop or keyboard interactions. If the list content is wider than the panel, a horizontal scrollbar appears.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/split-view/behavior-resize.png" />
+          </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+    </sky-demo-page-content>
+    <h3>Responsiveness</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>In smaller viewports, the split view automatically switches from a side-by-side view to a panel-switching interaction where users switch between the list and the workspace. The responsive workspace header should include iterator buttons for users to move between items without going back to the list.</p>
```

Comment:
I think "should include" should just be "includes," right? I didn't catch this the first time around, but this isn't a suggestion is it? This is just built-in functionality, I assume. If so, we should change "should include" to "includes"

Reply:
The iterators are no-longer built-in.

Reply:
How do users include the iterators? I don't think I saw an input in the reference docs for this.

Reply:
We originally had it as automatic portion of the workspace header but decided to pass on that functionality for MVP, as it adds quite a bit of complexity. Another approach we're discussing is to create an iterator component that users could include in their own templates. All that to say, is we haven't settled on a solution yet.

Reply:
In that case, should the docs include something like a recommendation for how to include the iterators? Or is that handled in the demo?

Reply:
As per our discussion in standup, I removed the sentence about including iterator buttons since we don't have a method for including them. ... However, I noticed that the image includes iterator buttons. @Blackbaud-ToddRoberts, can you remove the iterator buttons from that screenshot?

Reply:
I thought it was possible for the consumer to add them even if we don't have an iterator button component yet? We want the UX docs to represent what should be there even if it requires manual intervention.

Reply:
I thought from our discussion that it isn't possible to add iterator buttons, so we needed to update the docs to reflect that. However, if it is possible to add iterator buttons, my impression is that we'd need to update the docs to give some indication of how. @Blackbaud-AlexKingman, did I misunderstand? Can consumers add iterator buttons somehow?

Reply:
Example image is updated, sans iterator buttons